gcloud/rest/kms/client.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=duplicate-code
```

Comment:
Huh, I thought this was disabled globally?

Reply:
I got flagged for this when trying to commit. Will take a look once I commit again